### DISCUSSION TITLE
How to use vite-plugin-federation with svelte-kit

### DISCUSSION THREAD

**Discussion by gopala-blue** [originjs/vite-plugin-federation#125] [Q&A]
I have tried to integrate vite-plugin-federation with svelte-kit as svelte-kit uses vite. 
I had no luck integrating it. 

Facing issues like dynamic imports are not supported. 

Question:
Is it possible to integrate vite-plugin-federation with svelte-kit?

**Comment by Jiannan-dev** [comment]
I have not tried to integrate vite-plugin-federation under a framework like svelte(or nuxt...) This may require some additional work to make vite-plugin-federation support SSR frameworks
